### PR TITLE
[Term Entry] SwiftUI View Modifier: .font()

### LINKED ISSUE #1331: [Term Entry] SwiftUI ViewModifier: `.font()`
### Reason/inspiration (optional)

Please write a new entry under: content/swiftui/concepts/viewmodifier/terms/ about the modifier `.font()` in Swift.

- A "Syntax" section with pseudo code snippet. [Can also mention a note about custom font stacks]
- An "Example" section with a swift code snippet and image to accompany. Images should go in the media/ folder.

To get started with the PR and entry file:

Under content/swiftui/concepts/viewmodifier/terms/, create a new folder called `font`. Then create a new file called `font.md` under content/swiftui/concepts/viewmodifier/terms/`font`

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by gracekishino**
### Description

- Added new folder `font` inside `content/swiftui/concepts/viewmodifier/terms/`.
- Added new term entry file `content/swiftui/concepts/viewmodifier/terms/font/font.md` for SwiftUI ViewModifier: .font().
Added a new image for the SwiftUI ViewModifier: .font() term entry.

Closes #1331

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Review by SSwiniarski — APPROVED** [review]
@gracekishino This entry looks strong. I'm passing it right to a second review.

**Review by yangc95 — COMMENTED** [review]
@gracekishino Did a quick review!

**Comment by gracekishino** [comment]
@yangc95 thank you for the review and suggestions! I have made the changes.

File: content/swiftui/concepts/viewmodifier/terms/font/font.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    var body: some View {
+        Text("This text will have the font below applied to it.")
+            .font(.fontName)
+    }
+}
+```
+
+Inside `some View`, the `.fontName` font passed as an argument to the `.font()` modifier is applied to all text within the [text view](https://www.codecademy.com/resources/docs/swiftui/views/text). There are a selection of standard fonts available. It's also possible to create system fonts and add in custom fonts.
+
+Here are some standard fonts:
```
[the remark this answers is not in the document]
Comment (gracekishino) [reply content/swiftui/concepts/viewmodifier/terms/font/font.md:26]: @yangc95 do I need to do anything for this?

[the remark this answers is not in the document]
Comment (yangc95) [reply content/swiftui/concepts/viewmodifier/terms/font/font.md:26]: @gracekishino not at this time 🙂

**Review by yangc95 — APPROVED** [review]

**Comment by yangc95** [comment]
🥳 @gracekishino live link: https://www.codecademy.com/resources/docs/swiftui/viewmodifier/font

Screen Shot 2023-03-05 at 8 21 45 AM

cc: @SSwiniarski

**Pull request merged**